containers/layout/Layout.tsx

```diff
@@ -12,29 +12,27 @@ import { useAuth } from 'hooks/useAuth';
 import { useRouter } from 'next/router';
 import { FirebaseError } from 'lib/types/auth';
 import Error from 'components/error';
+import { withTranslation } from 'i18n.config';
+import { layoutVar } from 'lib/context-variables';
+import { useReactiveVar } from '@apollo/client';
 
 export interface LayoutProps {
-  home?: boolean;
-  heading?: string;
-  title?: string | string[];
-  error: FirebaseError | null;
   children?: React.ReactNode;
   i18n?: I18n;
   t: TFunction;
 }
 
-const Layout = ({
-  home = false,
-  error = null,
-  children,
-  heading,
-  title,
-  i18n,
-  t,
-}: LayoutProps): React.ReactElement => {
+const Layout = ({ children, t, i18n }: LayoutProps): React.ReactElement => {
```

Comment:
general question (nothing you did here): instead of passing around t and i18n, shouldn't we just use the useTranslation hook? All of a sudden, Layout would only have children prop, nothing else.

Reply:
Intuitively I would have used the `useTranslation` hook. But we didn't. So it could be that we did it this way for a reason. We need to check that out.